tools/rna_tools/inforna/inforna.xml

```diff
@@ -1,7 +1,7 @@
-<tool id="inforna" name="InfoRNA" version="0.1.0">
+<tool id="inforna" name="InfoRNA" version="2.1.2">
     <description></description>
     <requirements>
-        <requirement type="package" version="1.0">inforna</requirement>
+        <requirement type="package" version="2.1.2">inforna</requirement>
         <requirement type="package" version="2.1.5">vienna_rna</requirement>
```

Comment:
Ok, I will remove it.
I still need to make some test cases for this one. But  I'm checking if it will work.